src/ecsScene/scene.ts

```diff
@@ -50,7 +50,7 @@ function handleExternalAction(message: { type: string; payload: Record<string, a
           entity.remove(Gizmos)
         } else {
           const staticEntities = engine.getComponentGroup(StaticEntity)
-          if (!staticEntities.hasEntity(entity.uuid)) {
+          if (!staticEntities.hasEntity(entity)) {
```

Comment:
_Question_: Should this be here? I think it's a change on `decentraland-ecs`? but I don't see an update

Reply:
There is an update but my branch has the latest one, Juanca updated but removed the updated version so we can use mine. I think we should revert this.